src/lib/geo-location.js

```diff
@@ -27,6 +27,9 @@ const mockLocation = {
   subdivision: { code: 'CA', name: 'California' },
 }
 
+const isCacheEligible = (cacheObj, mode, country) =>
+  cacheObj !== undefined && mode === 'cache' && (cacheObj.data.country.code === country || !country)
```

Comment:
If you run `--country=XX`, shouldn't `mode` become `mock`? That's what the docs say is happening, and I wonder if reflecting that in the code would make this simpler, because then you wouldn't need to change this check at all?